post/models.py

```diff
@@ -26,6 +26,7 @@ class Comment(models.Model):
     name = models.CharField(max_length=100)
     body = models.TextField()
     date_added = models.DateTimeField(auto_now_add=True)
```

Comment:
Can you please change the naming of date_added and date_modified so that it fits the naming convention for Post model?

Reply:
sure , and done , will push soon